docs/tutorials/installing-macos-linux.rst

```diff
@@ -13,11 +13,13 @@ For installation instruction for Windows, see: :doc:`Installing on Windows <inst
 
 
     .. note::
-        You can skip this step if you are a Zapata employee and you already have an Orquestra account. Your Orquestra account will give you access to Zapata's Nexus repository. If you are a Zapata employee without an Orquestra account, you can request an account through the `Zapata Cloud Support portal <https://zapatacomputing.atlassian.net/servicedesk/customer/portal/4/group/14/create/35>`_.
+        You can skip this step if you are a Zapata employee and you already have an Orquestra account.
+        Your Orquestra account will give you access to Zapata's Nexus repository.
+        If you are a Zapata employee without an Orquestra account, you can request an account through the `Zapata Cloud Support portal <https://zapatacomputing.atlassian.net/servicedesk/customer/portal/4/group/14/create/35>`_.
 
   Update your pip configuration file
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏This references old Nexus.

Is it needed? Can we get rid of it?

Reply:
Is there a SSOT we can point to instead?

Reply:
We have internal Nexus documentation, but it's not needed for `orquestra-sdk` since it's on pypi

Reply:
Good point! Remove it entirely? [58d470d](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/292/commits/58d470d2f437412f669548175f052c24f82aa786)?